src/plugins/default/markbind-plugin-anchors.js

```diff
@@ -2,39 +2,25 @@ const cheerio = module.parent.require('cheerio');
 const md = require('./../../lib/markbind/src/lib/markdown-it');
 
 const ANCHOR_HTML = '<a class="fa fa-anchor" href="#"></a>';
-
-/**
- * Generates a heading selector based on the indexing level
- * @param headingIndexingLevel to generate
- */
-function generateHeadingSelector(headingIndexingLevel) {
-  let headingsSelector = 'h1';
-  for (let i = 2; i <= headingIndexingLevel; i += 1) {
-    headingsSelector += `, h${i}`;
-  }
-  return headingsSelector;
-}
-
+const HEADER_TAGS = 'h1, h2, h3, h4, h5, h6';
 /**
  * Adds anchor links to headers
  */
 module.exports = {
+  // eslint-disable-next-line no-unused-vars
   postRender: (content, pluginContext, frontMatter, pageConfig) => {
```

Comment:
After looking through `filterTags`, I think that we should maintain consistency between our in-built plugins. 

Therefore, we should remove the unused variables `pluginContext`, `frontMatter` and `pageConfig` rather than using the eslint disable next line.